Gopkg.lock

```diff
@@ -2111,7 +2111,7 @@
   revision = "b3a7cee44a305be0a69e1b9ac03018307287e1b0"
 
 [[projects]]
-  digest = "1:f27698f7ae7864893ebcfb843e44d821263ac1dcf0ba1d5c2353f9d319a2f28d"
+  digest = "0:"
```

Comment:
Do you know why this changed ?

Reply:
to be honest, I don't. It was failing CI though, saying `Gopkg.lock` was out of sync and that I should run `inv -e deps`.
My _hunch_ is that `netlink` was just a transitive dependency before (`go-conntrack` depends on `netlink`), but now I'm directly referencing the package.